packages/core/src/client.ts

```diff
@@ -666,6 +666,10 @@ export class MedplumClient extends EventTarget {
       this.refreshProfile().catch(console.log);
     }
 
+    if (options?.accessToken) {
+      this.setAccessToken(options.accessToken);
+    }
```

Comment:
Yep, this is the meat of it.

I think we should move lines 663-667 into an `else` block though, to avoid confusion.  If user passes in an `accessToken`, we're going to ignore existing login.

In practice, this is unlikely to matter too much (the use cases are pretty different), but I'd prefer to be explicit.